pagopa/io-app
# feat: [IOCOM-866] Display tags in the header of a SEND message

This PR depends on https://github.com/pagopa/io-app/pull/5431

## Short description
This PR updates the `MessageDetails` component in order to display the tags.

_NOTE: Other PRs will follow to update the rest of the message._

<details open><summary>Details</summary>
<p>

| MessageDetails |
| - |
| <img src="https://github.com/pagopa/io-app/assets/29163287/3df211d0-43fa-460a-b7a7-b2ef0a514f41" width="300"/> |

</p>
</details> 

## List of changes proposed in this pull request
- added the tags in the header of the SEND message

## How to test
Using the `io-dev-api-server`, generate a SEND message. Enable the design system and navigate to the message details. Check that the content is displayed correctly.
